src/introduction/a-path-to-pos-zcash.md

```diff
@@ -14,35 +14,58 @@ We envision a *proof-of-stake transition path* as one of the potential paths wit
 
 ## A Proof-of-Stake Transition Path
 
-Given that context, we envision a "path" within the Zcash Tech-Tree for transitioning Zcash to PoS. At the top level we propose that this path contain at least two major milestones:
+Given that context, we envision a path within the Zcash Tech-Tree for transitioning Zcash to PoS. At the top level we propose that this path contain at least two major milestones:
 
-1. Transitioning from current Zcash PoW to a hybrid PoW/PoS system.
-2. Transitioning from a hybrid PoW/PoS system to pure PoS.
+1. Transitioning from current Zcash [NU5](../terminology.md#definition-nu5) [PoW](../terminology.md#definition-pow) protocol to a PoW/PoS [hybrid consensus](../terminology.md#definition-hybrid-consensus) protocol dubbed [PoW+TFL](../terminology.md#definition-pow-tfl).
+2. Transitioning from [PoW+TFL](../terminology.md#definition-pow-tfl) to pure [PoS](../terminology.md#definition-pow-tfl) protocol.
 
 After this transition to pure PoS, there are likely to be future improvements to the PoS protocol, or the consensus protocol more generally. This TFL book focuses almost exclusively on the first step in this sequence.
 
 Our primary motivation for proposing (at least) two steps is to minimize disruption to usability, safety, security, and the ecosystem during each step.
 
+This book primarily focuses this first step: the transition to [PoW+TFL](../terminology.md#definition-pow-tfl). To understand the specific goals for that, see [Design Goals](../overview/design-goals.md). 
+
 With this approach, the Zcash Tech Tree with the [TFL](../terminology.md#definition-tfl) approach might look something like this:
 
 ```dot process
 {{#include ../diagrams/zcash-tech-tree-tfl-steps.dot}}
 ```
 
-## Design Goals for a Hybrid PoW/PoS System
+## Why Two Steps?
+
+One question we've gotten in proposing this approach is why take a two step process with an intermediate [hybrid consensus](../terminology.md#definition-hybrid-consensus) protocol, rather than a single transition directly to a [PoS](../terminology.md#definition-pos) protocol? 
```

Comment:
```suggestion
One question we've gotten in proposing this approach is why take a two-step process with an intermediate [hybrid consensus](../terminology.md#definition-hybrid-consensus) protocol, rather than a single transition directly to a [PoS](../terminology.md#definition-pos) protocol? 
```